lib/code-examples.js

```diff
@@ -1,109 +1,124 @@
 export default {
   'basic-interval': {
     name: 'Basic interval',
-    code: `Rx.Observable
-  .interval(1000)
-  .take(4)
+    code: `const { interval } = Rx.sources;
+const { take } = Rx.operators;
+
+interval(1000).pipe(take(4));
 `,
     timeWindow: 5000
   },
   'random-error': {
     name: 'Random error',
-    code: `Rx.Observable
-  .create(observer => {
-    let n = 1;
-
-    const intervalId = setInterval(() => {
-      if (Math.random() < 0.8 && n < 9) {
-        observer.next(n * n);
-        n += 1;
-      } else {
-        observer.error('Oh no...');
-      }
-    }, 1000);
-
-    return () => clearInterval(intervalId);
+    code: `const { Observable } = Rx.sources;
+const { take } = Rx.operators;
+Observable.create(observer => {
+  let n = 1;
+  const intervalId = setInterval(() => {
+    if (Math.random() < 0.8 && n < 9) {
+      observer.next(n * n);
+      n += 1;
+    } else {
+      observer.error('Oh no...');
+    }
+  }, 1000);
+
+  return () => clearInterval(intervalId);
   })
 `,
     timeWindow: 10000
   },
   'chess-game': {
     name: 'Chess game',
-    code: `const timer$ = Rx.Observable.interval(1000);
-const pieces$ = Rx.Observable.of('', '♞', '', '♞', '♘', '♞');
-const columns$ = Rx.Observable.of('e', 'c', 'g', 'd', 'e', 'f');
-const rows$ = Rx.Observable.of('4', '6', '4', '4', '2', '3');
-
-Rx.Observable.zip(
-  timer$,
-  pieces$,
-  columns$,
-  rows$,
-  (_, piece, column, row) => \`\${piece}\${column}\${row}\`
-)
+    code: `const { of, interval } = Rx.sources;
+const { zip, map } = Rx.operators;
+const timer$ = interval(1000);
+const pieces$ = of('', '♞', '', '♞', '♘', '♞');
+const columns$ = of('e', 'c', 'g', 'd', 'e', 'f');
+const rows$ = of('4', '6', '4', '4', '2', '3');
+timer$.pipe(
+  zip(
+    pieces$,
+    columns$,
+    rows$
+  ),
+  map(([timer, piece, column, row]) => {
+    return \`\${piece}\${column}\${row}\`
+  })
```

Comment:
Is there a good reason to use `map` instead of passing the mapping function to `zip` like before?

Reply:
The mapping function is deprecated starting at v6 and will be removed in v7 so I thought I'd do the switch from now. [Link.](https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/v6/migration.md#howto-result-selector-migration)
The reason for that if I recall correctly is to simplify the type definitions.